spec/models/manageiq/providers/kubernetes/container_manager_spec.rb

```diff
@@ -47,4 +47,40 @@
       expect(job.zone).to eq("default")
     end
   end
+
+  context "Proxy Configuration" do
```

Comment:
@chrisarcand Ok, this make sense to me. I will change the test to be similar to what you suggested above, Although I don't understand how to guess the return value from ```VMDB::Util.http_proxy_uri```.